server/client.go

```diff
@@ -370,15 +371,13 @@ var keyOrIDRegex = regexp.MustCompile("(^[[:alpha:]]+-)?[[:digit:]]+$")
 func endpointNameFromRequest(r *http.Request) string {
 	_, path := splitInstancePath(r.URL.Path)
 	l := strings.ToLower(path)
-	s := strings.TrimLeft(l, "/rest/api")
-	if s == l {
-		return "_unrecognized"
-	}
```

Comment:
Why was this case removed?

Reply:
When I changed from TrimLeft to TrimPrefix I broke the tests and it needed a bit of tweaking to work, and after finding the actual problem I never noticed I could re-add it. I've fixed it now, thanks.